src/testdir/test_cmdline.vim

```diff
@@ -1716,6 +1716,7 @@ func Test_verbose_option()
     command DoSomething echo 'hello' |set ts=4 |let v = '123' |echo v
     call feedkeys("\r", 't') " for the hit-enter prompt
     set verbose=20
+    set messagesopt&
```

Comment:
Why is this added here?

Reply:
Sorry.  It is mistake.  I will remove it later.